src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx

```diff
@@ -190,11 +192,16 @@ export const MintingDepositTimeline: FC<MintingTimelineProps> = ({
       <BodyLg
         color="hsl(0, 0%, 50%)"
         fontWeight="medium"
-        mb={isOnDepositDetailsPage || account ? 10 : 6}
+        mb={isOnDepositDetailsPage ? 2 : account ? 10 : 6}
         lineHeight={6}
       >
         {title}
       </BodyLg>
+      {isOnDepositDetailsPage && (
+        <LabeledBadge icon={ClockIcon} label="Elapsed time" mb={10}>
+          3 hrs
```

Comment:
Non-blocking but good to have:

We should add a TODO here. This value should be calculated based on the time where the reveal was done (but in a separate PR). I think we've done a similar thing in token-dashbard for redemption. 

We can also create a task for it.

Reply:
Are you talking about this [Badge](https://github.com/threshold-network/token-dashboard/blob/265f38948840e745171b6368e28942f96be8197e/src/pages/tBTC/Bridge/DepositDetails.tsx#L252C1-L262C25)?
It's hardcoded in token-dashboard

Reply:
Added TODO comment: 
Ref commit: bb0d020fbedf3638cb5ee907d16c18562a6ed7c3